buckup/bucket_creator.py

```diff
@@ -125,19 +126,19 @@ def set_bucket_policy(self, bucket, user, public_get_object_paths=None):
                 break
         print('Bucket policy set.')
 
-        if public_access:
-            # NB: This API doesn't exist on a `Bucket`
-            self.s3_client.put_public_access_block(
-                Bucket=bucket.name,
-                # Allow policies to provide access to objects, but not ACLs
-                PublicAccessBlockConfiguration={
-                    "BlockPublicAcls": True,
-                    "IgnorePublicAcls": True,
-                    "BlockPublicPolicy": False,
-                    "RestrictPublicBuckets": False
-                }
-            )
-            print('Enabled public access to the bucket.')
+        # NB: This API doesn't exist on a `Bucket`
+        self.s3_client.put_public_access_block(
+            Bucket=bucket.name,
+            PublicAccessBlockConfiguration={
+                # Prevent files being marked as public using ACLs
```

Comment:
suggestion: Remove or update this comment so it describes all the possible states.